Tests/NIOFSTests/FileHandleTests.swift

```diff
@@ -170,15 +175,23 @@ internal final class FileHandleTests: XCTestCase {
     }
 
     func testValueForAttribute() async throws {
-        var nonThrowingErrnos: [Errno] = [.noData]
+        var nonThrowingErrnos: [Errno] = []
         var knownErrnos: [Errno: FileSystemError.Code] = [.notSupported: .unsupported]
-        #if canImport(Darwin)
+        #if canImport(Darwin) || os(FreeBSD)
```

Comment:
This represents a behavioural change on macOS, which no longer has `noData` in its `nonThrowingErrnos`.